stl/inc/print

```diff
@@ -0,0 +1,227 @@
+// print standard header
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef _PRINT_
+#define _PRINT_
+#include <yvals_core.h>
+#if _STL_COMPILER_PREPROCESSOR
+#ifndef __cpp_lib_print
+_EMIT_STL_WARNING(STL4038, "The contents of <print> are available only with C++23 or later.");
+#else // ^^^ !defined(__cpp_lib_print) / defined(__cpp_lib_print) vvv
+
+#include <__msvc_print.hpp>
+#include <chrono>
+#include <cstdio>
+#include <format>
+#include <string>
+
+#pragma pack(push, _CRT_PACKING)
+#pragma warning(push, _STL_WARNING_LEVEL)
+#pragma warning(disable : _STL_DISABLED_WARNINGS)
+_STL_DISABLE_CLANG_WARNINGS
+#pragma push_macro("new")
+#undef new
+
+_STD_BEGIN
+
+inline void _Print_noformat_nonunicode(FILE* const _Stream, const string_view _Str) {
+    const bool _Was_write_successful = (_STD fputs(_Str.data(), _Stream) != EOF);

```

Comment:
This is a buffer overrun; `_Str.data()` is not required to be nul-terminated, but `fputs` expects a nul-terminated byte string.

Reply:
This should likely use `fwrite`:

```suggestion
    const size_t _Bytes_written = _CSTD fwrite(_Str.data(), 1, _Str.size(), _Stream);
```